library/CHART-TEMPLATE/Chart.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v2
+name: CHART-TEMPLATE
```

Comment:
the name is uppercase to avoid it to be installable?

Reply:
It is uppercase to be easier to `grep CHART-TEMPLATE` or `sed -i s_CHART-TEMPLATE_whatever_` and to catch attention while templating it.

Reply:
fine for me!